docs/user/integrations.rst

```diff
@@ -1,226 +1,50 @@
-inside Git_, Subversion_, Mercurial_, or Bazaar_.
-
-.. _Git: http://www.kernel.org/pub/software/scm/git/docs/githooks.html
-.. _Subversion: https://www.mikewest.org/2006/06/subversion-post-commit-hooks-101
-.. _Mercurial: http://hgbook.red-bean.com/read/handling-repository-events-with-hooks.html
-.. _Bazaar: http://wiki.bazaar.canonical.com/BzrHooks
-
-Authentication
-~~~~~~~~~~~~~~
-
-This endpoint requires authentication. If authenticating with an integration
-token, a check will determine if the token is valid and matches the given
-project. If instead an authenticated user is used to make this request, a check
-will be performed to ensure the authenticated user is an owner of the project.
-
-Debugging webhooks
-------------------
-
-If you are experiencing problems with an existing webhook, you may be able to
-use the integration detail page to help debug the issue. Each project
-integration, such as a webhook or the generic API endpoint, stores the HTTP
-exchange that takes place between Read the Docs and the external source. You'll
-find a list of these exchanges in any of the integration detail pages.
-
-Resyncing webhooks
-------------------
+The Continuous Integration and Continuous Deployment (CI/CD) features are configured with your repository provider,
+such as GitHub, Bitbucket or GitLab,
+and with each change committed to your repository, Read the Docs is notified by the webhook.
 
-It might be necessary to re-establish a webhook if you are noticing problems.
-To resync a webhook from Read the Docs, visit the integration detail page and
-follow the directions for re-syncing your repository webhook.
+When we receive an integration notification, we determine if the change is related to an active version for your project.
+If it is, a build is triggered for that version.
 
-Payload validation
-------------------
+Continuous Documentation for software projects
+----------------------------------------------
 
-If your project was imported through a connected account,
-we create a secret for every integration that offers a way to verify that a webhook request is legitimate.
-Currently, `GitHub <https://developer.github.com/webhooks/securing/>`__ and `GitLab <https://docs.gitlab.com/ee/user/project/integrations/webhooks.html#validate-payloads-by-using-a-secret-token>`__
-offer a way to check this.
+Documentation fits into any CI/CD pipeline through the idea and objectives of *Documentation as Code*.
+A straight-forward method to achieve this is by maintaining documentation alongside the source code,
+meaning that the documentation's life cycle is contained within the same git repository as the source code.
 
-Troubleshooting
----------------
+When changes happen to the source code, changes should also happen to the documentation.
+By managing these changes in the same life-cycle,
+you can benefit from **documentation and source code being part of the same code review process**.
```

Comment:
Flagging for Tuesday: I'd like to add Versioning in quick succession after this.